CONTRIBUTING.md

```diff
@@ -77,6 +77,44 @@ README for more details.
 - Once a pull request is good to go, the code reviewer is the one who merges
 it, then deletes the branch.
 
+## Front end architecture
+### CSS
+- The front end will use the Web Design Standard's components, imported through ruby, adding extra functionality that's missing.
+- Atomic CSS methodolgy will be used for layout, modifier classes.
```

Comment:
atomic css handles more than just layout; would rephrase to:

"The front end utilizes single-purposed, tightly scoped atomic classes to build new UI"